src/commands/General/guide.ts

```diff
@@ -1,53 +1,71 @@
-import { WebsiteUrl, docsButtonBuilder, inviteSupportDiscordButton } from '#lib/components/button';
+import { getSupportedUserLanguageT } from '#lib/i18n/translate';
 import { BirthdayyCommand } from '#lib/structures';
 import { ConfigApplicationCommandMentions } from '#root/commands/Admin/config';
 import { BirthdayApplicationCommandMentions } from '#root/commands/Birthday/birthday';
-import { Emojis } from '#utils/constants';
-import { defaultEmbed } from '#utils/embed';
-import { CLIENT_NAME } from '#utils/environment';
-import { ApplicationCommandRegistry, container } from '@sapphire/framework';
-import { applyLocalizedBuilder, resolveKey, type Target } from '@sapphire/plugin-i18next';
-import { ActionRowBuilder, ButtonBuilder, EmbedBuilder } from 'discord.js';
+import { BrandingColors, Emojis } from '#utils/constants';
+import { ApplicationCommandRegistry } from '@sapphire/framework';
+import { applyLocalizedBuilder, type TFunction } from '@sapphire/plugin-i18next';
+import { ActionRowBuilder, ButtonBuilder, ButtonStyle, EmbedBuilder } from 'discord.js';
 
 export class GuideCommand extends BirthdayyCommand {
 	public override async registerApplicationCommands(registry: ApplicationCommandRegistry) {
 		registry.registerChatInputCommand((builder) =>
-			applyLocalizedBuilder(builder, 'commands/guide:guide').setDMPermission(true)
+			applyLocalizedBuilder(builder, 'commands/guide:name', 'commands/guide:description').setDMPermission(true)
```

Comment:
did we agree on having the command names hardcoded in english? I remember something about this but I am not sure!

Reply:
Oh, would you like to do it on all commands? 😬

Reply:
> Oh, would you like to do it on all commands? 😬

I think so yes, so that only the description of the command is translated, not the command itself!

Reply:
> > Oh, would you like to do it on all commands? 😬
> 
> I think so yes, so that only the description of the command is translated, not the command itself!

I confess that I'm a little mixed with this idea, why not make it a poll on discord?

Reply:
Its about so that there are not multiple different names for the same thing. 
Everybody, no matter what language you are speaking knows what a "birthday" or a "config" is in english.
Command description, attributes and label can be in the users language, just the exact name of the command not.

Like this
![CleanShot 2024-05-13 at 11 34 02@2x](https://github.com/BirthdayyBot/BirthdayyBot/assets/58025175/0a087ee4-f192-4d5c-9144-33b194c54106)


This is a user experience decision not something the public should vote about in my eyes!

Reply:
I understand your point of view,I would let you do a PR on this subject in this case 👍🏼

Reply:
will do!